packages/spatial/src/camera/components/FollowCameraComponent.ts

```diff
@@ -213,7 +213,7 @@ const computeCameraFollow = (cameraEntity: Entity, referenceEntity: Entity) => {
   const follow = getComponent(cameraEntity, FollowCameraComponent)
   const followState = getMutableComponent(cameraEntity, FollowCameraComponent)
   const cameraTransform = getComponent(cameraEntity, TransformComponent)
-  const targetTransform = getComponent(referenceEntity, TransformComponent)
+  const targetTransform = getOptionalComponent(referenceEntity, TransformComponent)
```

Comment:
what was this breaking for?

Reply:
1000s of warning messages in FollowCameraComponent from the Transform being computed. Not sure where it came from. Because this code hasn't changed.